playground/nextjs/app/app-dir/sign-up/page.tsx

```diff
@@ -1,5 +1,5 @@
 import { SignUp } from '@clerk/nextjs/app-beta';
 
 export default function Page() {
-  return <SignUp />;
+  return <SignUp unsafeMetadata={{ playground: 'nextjs', type: 'app-dir' }} />;
```

Comment:
:upside_down_face: Not sure if we should commit these changes in our playground apps. It doesn't hurt, just uncertain if this feature deserves much spotlight.

Reply:
@gkats I left it in as a sort of litmus test for the future. Happy to remove them if we feel they shouldn't be there.

Reply:
I'd say let's remove them until we figure out which features we want to showcase in the playground apps. :wink:

Reply:
Removed! 👍